api/envoy/admin/v2alpha/BUILD

```diff
@@ -1,7 +1,18 @@
-load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library_internal")
+load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library_internal", "api_proto_package")
 
 licenses(["notice"])  # Apache 2
 
+api_proto_package(
+    deps = [
```

Comment:
When do you have default vs. explicit `name = "v2",`?

Reply:
I started naming them starting from envoy/api/v2/core, and then switched to using the default name. I'm fine making them all to be `pkg`. I thought `envoy/api/v2:v2` looked better at first.